pkg/manager/interfaces/reservation.go

```diff
@@ -0,0 +1,13 @@
+package interfaces
+
+import (
+	"context"
+
+	"github.com/flyteorg/flyteidl/gen/pb-go/flyteidl/datacatalog"
+)
+
+type ReservationManager interface {
```

Comment:
Mind adding a comment describing what this public interface does?

Reply:
Will add docs once we are finalized on the interface..